development-workflow.md

```diff
@@ -0,0 +1,61 @@
+# MeiliSearch Development Workflow
+
+MeiliSearch is an open soure project. 
+
+This document provides a general overview of the workflow for its development. 
+
+This document covers only the workflow for the [main project](meilisearch/MeiliSearch).
+
+[TOC]
+
+## Governance
+
+### MeiliSearch and Meili (the company)
+MeiliSearch is supported and developed mostly by Meili, the company. The project aims to be more and more managed by the community at large. Teams membership are, in theory, open to anyone working, or not, for Meili, the company.
+
+> __Disclaimer__
+> _The description of the teams and governance in this document are completely unrelated to how the company works and is organized internally._
+
+## 🔮 Roadmap
+The MeiliSearch roadmap is public on [roadmap.meilisearch.com](https://roadmap.meilisearch.com).
+Anyone can vote for future features or submit an new idea. The roadmap is moderated and prioritized by MeiliSearch CEO.
+When an idea on the roadmap is prioritized it will enter in the development workflow.
+
+> __Note__
+> Not all specifications must be born out of a card on the roadmap and not all roadmap cards will be transformed into specifications.
```

Comment:
Isn't the latest the goal? To be really open and public?

Reply:
No. While MeiliSearch is mainly supported by Meili, the roadmap will be influenced by the business and it would be dishonest to say the opposite.
We will try our best in making this public and transparent but we will have prioritization that can not always be transparent for business & strategic reasons.

Reply:
ok